src/Psy/ColorMode.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/*
+ * This file is part of Psy Shell.
+ *
+ * (c) 2012-2015 Justin Hileman
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Psy;
+
+/**
+ * A configuration object that represents one of 3 available color modes:
+ *   1. "auto" (the default): if color support is detected, output contains colors
+ *   2. "forced": force colors in output
+ *   3. "disabled": disable colors in output
+ */
+class ColorMode
```

Comment:
TBH I don't see the advantage of using a class for this over just using three constants and checking equality in services that consume it. Convince me?